src/components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Title from './Title';
+import Colors from '../../variables';
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  background-color: ${(props) =>
+    props.darkmode ? Colors.DarkBlue : Colors.Beige};
+  border: 1px solid white;
+`;
+
+const Form = ({ children, text, darkmode }) => {
+  return (
+    <div>
+      <StyledForm darkmode={darkmode}>
+        <Title text={text} />
+        {children}
+      </StyledForm>
+    </div>
+  );
+};
+
+Form.propTypes = {
+  children: PropTypes.element,
+  text: String,
+  darkmode: true,
+};
```

Comment:
```suggestion
Form.propTypes = {
  children: PropTypes.element,
  text: String,
  darkmode: Boolean,
};
```
small change here, should be set to the datatype instead of the value true